windows/src/engine/keyman/UfrmKeyman7Main.pas

```diff
@@ -1279,6 +1284,22 @@ procedure TfrmKeyman7Main.WndProc(var Message: TMessage);
 end;
 
 procedure TfrmKeyman7Main.ProcessProfileChange(CommandAndAtom: DWORD);   // I3933   // I3949
+
+  procedure ProcessHotkeyChange(hkl: DWORD; guidProfile: TGUID);
+  var
+    kbd: TLangSwitchKeyboard;
+  begin
+    kbd := FLangSwitchManager.FindKeyboard(hkl, guidProfile);
+    if Assigned(kbd) then
+    begin
+      // Handle toggle hotkey
+      if (kbd = FLangSwitchManager.ActiveKeyboard) and (kmcom.Options['koKeyboardHotkeysAreToggle'].Value) then
+          kbd := FLangSwitchManager.Languages[0].Keyboards[0];
```

Comment:
Ah, this is toggling the Keyman keyboard off into the default keyboard layout.

So if the default keyboard is selected (not `Assigned(kbd)`), will this also toggle into the Keyman keyboard?

Reply:
`FindKeyboard` returns `nil` (i.e. `not Assigned`) if no keyboard can be found that matches the hotkey. Even if the default keyboard is currently selected, `FindKeyboard` will still return the keyboard associated with hotkey, so the default keyboard won't be selected